skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -165,6 +167,11 @@ public function executeIndex( $params ) {
 		// check for video add button permissions
 		$this->showAddVideoBtn = $wgUser->isAllowed( 'videoupload' );
 
+		$this->showCuratedContentToolBtn =
+			WikiaPageType::isMainPage() &&
+			!empty( $wgEnableCuratedContentExt ) &&
```

Comment:
maybe we should consider checking this wg variable in mercury too because we are asking controller directly there and if it's set to false it would throw an error